### PR TITLE
update readme with consumer relationships

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,12 @@ This library is the canonical location of the Open Cluster Management API defini
 
 ## Consumers
 
-Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. For example, see [registration-operator](https://github.com/open-cluster-management-io/registration-operator) for how the `api` is being consumed.
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. The following are relationships between api and consumers.
+*  `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim`: implemented in [registration](https://github.com/open-cluster-management-io/registration).
```
Comment (qiujian16) [inline README.md]: registration: implements xxxx for cluster registration and lifecycling.

Comment (xuezhaojun) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,13 @@ This library is the canonical location of the Open Cluster Management API defini
...
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. 
+
+* [registration](https://github.com/open-cluster-management-io/registration): implements `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim` for cluster registration and lifecycling.
+* [work](https://github.com/open-cluster-management-io/work): implements `Manifestwork` for kubernetes resource distribution 
+* [addon-framework](https://github.com/open-cluster-management-io/addon-framework): implements `ClusterManagementAddOn`, `ManagedClusterAddOn` for addon management.
+* [placement](https://github.com/open-cluster-management-io/placement): implements `Placement` for placement of policies to managed clusters.
+* [registration-operator](https://github.com/open-cluster-management-io/registration-operator): implements `ClusterManager`, `Klusterlet` for deployment of components including registration, work, addon-framework, etc.
```
Comment (qiujian16) [inline README.md]: for deployment of components including registration, work, addon-framework, etc. --》
as an operator to deploy registration, work and placement

File: README.md [hunk 3]
```diff
@@ -15,7 +15,13 @@ This library is the canonical location of the Open Cluster Management API defini
 
 ## Consumers
 
-Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. For example, see [registration-operator](https://github.com/open-cluster-management-io/registration-operator) for how the `api` is being consumed.
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. 
+
+* [registration](https://github.com/open-cluster-management-io/registration): implements `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim` for cluster registration and lifecycling.
+* [work](https://github.com/open-cluster-management-io/work): implements `Manifestwork` for kubernetes resource distribution 
+* [addon-framework](https://github.com/open-cluster-management-io/addon-framework): implements `ClusterManagementAddOn`, `ManagedClusterAddOn` for addon management.
+* [placement](https://github.com/open-cluster-management-io/placement): implements `Placement` for placement of policies to managed clusters.
```
Comment (qiujian16) [inline README.md]: for placement of policies to managed clusters. --> for cluster selection with various policies to deploy workloads.

File: README.md [hunk 4]
```diff
@@ -15,7 +15,13 @@ This library is the canonical location of the Open Cluster Management API defini
 
 ## Consumers
 
-Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. For example, see [registration-operator](https://github.com/open-cluster-management-io/registration-operator) for how the `api` is being consumed.
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. 
+
+* [registration](https://github.com/open-cluster-management-io/registration): implements `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim` for cluster registration and lifecycling.
+* [work](https://github.com/open-cluster-management-io/work): implements `Manifestwork` for kubernetes resource distribution
```
Comment (qiujian16) [inline README.md]: for native kubernetes resource distribution to multiple clusters.

File: README.md (hunk 2 above)
Comment (xuezhaojun) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (xuezhaojun) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (xuezhaojun) [reply README.md]: Done

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**